# Only in ./web/modules/custom/cig_pods: cig_pods.info.yml
# Only in ./web/modules/custom/cig_pods: cig_pods.install
# Only in ./web/modules/custom/cig_pods: cig_pods.libraries.yml
# Only in ./web/modules/custom/cig_pods: cig_pods.links.action.yml
# Only in ./web/modules/custom/cig_pods: cig_pods.module
# Only in ./web/modules/custom/cig_pods: cig_pods.post_update.php
# Only in ./web/modules/custom/cig_pods: cig_pods.routing.yml
# Only in ./web/modules/custom/cig_pods: cig_pods.views.inc
# Only in ./web/modules/custom/cig_pods_csc: cig_pods_csc.info.yml
# Only in ./web/modules/custom/cig_pods_csc: cig_pods_csc.routing.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.award.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.awardee.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.contact.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.cost_sequence.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: asset.type.csc_field_enrollment.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: asset.type.csc_import_history.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: asset.type.csc_partner_activities.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: asset.type.csc_producer_enrollment.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: asset.type.csc_project.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: asset.type.csc_project_summary.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.field_assessment.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.input.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.irrigation.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.lab_result.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.lab_testing_method.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.operation.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.pasture_assessment.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.pasture_health_assessment.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.producer.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.project.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.range_assessment.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.shmu_crop_rotation.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.soil_health_management_unit.yml
# Only in ./web/modules/custom/cig_pods/config/install: asset.type.soil_health_sample.yml
# Only in ./web/modules/custom/cig_pods/config/install: farm_map.map_behavior.latlon_add.yml
# Only in ./web/modules/custom/cig_pods/config/install: farm_map.map_behavior.wkt_refresh.yml
# Only in ./web/modules/custom/cig_pods/config/install: farm_map.map_behavior.wkt_refresh_soil_sample.yml
# Only in ./web/modules/custom/cig_pods/config/install: farm_map.map_behavior.zoom_us.yml
# Only in ./web/modules/custom/cig_pods/config/install: farm_map.map_type.pods.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.csc_alley_cropping.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_anaerobic_digester.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_combustion_sys_improvement.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_conservation_cover.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_conservation_crop_rotation.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_contour_buffer_strips.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_cover_crop.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_critical_area_planting.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_feed_management.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_field_border.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_filter_strip.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_forest_farming.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_forest_stand_improvement.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_grassed_waterway.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_hedgerow_planting.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_herbaceous_wind_barriers.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_market_activities.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_mulching.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_nutrient_management.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_pasture_hay_planting.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_prescribed_grazing.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_range_planting.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_residue_till_reduced_till.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_riparian_forest_buffer.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_riparian_herbaceous_cover.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_roofs_and_covers.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_silvopasture.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_stripcropping.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_tree_shrub_establishment.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_vegetative_barrier.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_waste_separation_facility.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_waste_storage_facility.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_waste_treatment.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_waste_treatment_lagoon.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.csc_windbreak_shelterbelt.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.environmental_benefits.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.farm_summary.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.field_summary.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.ghg_benefits_alternate_modeled.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: log.type.ghg_benefits_measured.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.666_implementation_purpose.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.activity_by_partner.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.avoided_land_conversion.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.avoided_land_conversion_amount_u.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.avoided_land_conversion_purpose.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.baseline_yield_location.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.baseline_yield_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.commodity_category.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.commodity_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.cost_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.county.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.csaf_experience.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.csaf_federal_funds.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.csaf_market_incentives.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.csaf_nonprofit_funds.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.csaf_state_or_local_funds.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_aggregate_stability_me.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_aggregate_stability_un.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_contact_type.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_cost_type.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_cover_crop.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_crop.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_ec_method.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_equipment.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_equipment_ownership.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_experimental_design.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_grant_type.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_grazing_type.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_infiltration_method.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_input.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_lab_interpretation.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_laboratory.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_land_use.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_land_use_modifiers.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_major_resource_concern.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_nitrate_n_method.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_operation_type.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_ph_method.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_plant_stage.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_practice.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_resource_concern.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_respiration_detection_.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_respiration_incubation.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_shmu_type.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_soil_health_extraction.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_state_territory.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_tillage_system.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_tractor_self_propelled_machine.yml
# Only in ./web/modules/custom/cig_pods/config/install: taxonomy.vocabulary.d_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.environmental_benefits.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_any_csaf_practice.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_commodity_volume_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_ghg_calculations.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_ghg_monitoring.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_ghg_reporting.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_ghg_verification.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_irrigated.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_land_use.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_official_ghg_calculation.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.field_tillage.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.fuel_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_calculation_methods.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_cumulative_calculation.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_measurement_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_model.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_monitoring_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_reporting_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.ghg_verification_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.improved_wildlife_habitat.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.improved_wildlife_habitat_amount.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.improved_wildlife_habitat_purpos.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.incentive_per_acre_or_head.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.incentive_reason.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.incentive_structure.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.incentive_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.livestock_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.log_category.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.marketing_assistance_provided.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.marketing_channel_geography.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.marketing_channel_identification.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.marketing_channel_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.marketing_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.match_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.measurement_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.mmrv_assistance_provided.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.new_partnership.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.nutrient_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.organic_farm.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.organic_fields.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.other_field_measurement.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.other_water_quality.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.other_water_quality_amount_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.other_water_quality_purpose.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.other_water_quality_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.partner_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.payment_on_enrollment.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.payment_on_harvest.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.payment_on_implementation.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.payment_on_mmrv.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.payment_on_sale.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.practice_extent_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.practice_past_extent.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.practice_standard.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.practice_this_field.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.practice_type.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.price_premium_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.producer_motivation.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.producer_outreach.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.producer_ta_received.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.product_differentiation_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_energy_use.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_energy_use_amount_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_energy_use_purpose.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_erosion.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_erosion_amount_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_erosion_purpose.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduced_phosphorus_loss_amount_u.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduction_in_nitrogen_loss.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduction_in_nitrogen_loss_amoun.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduction_in_nitrogen_loss_purpo.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduction_in_phosphorus_loss.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.reduction_in_phosphorus_loss_pur.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.soil_sample_result_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.state.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.total_area.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.traceability_method.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.training_provided.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.underserved_status.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.volume_sold_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.waste_storage_system.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.water_quantity.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.water_quantity_amount_unit.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: taxonomy.vocabulary.water_quantity_purpose.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: user.role.admin.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: user.role.analyst.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: user.role.awardee.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.alley_cropping_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.anerobic_digester_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.combustion_system_improvement_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.conservation_cover_crop_rotation_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.conservation_cover_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.contour_buffer_strips_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.cover_view.yml.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.critical_area_planting_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.csc_field_enrollment.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.environmental_benefits_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.farm_summary_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.feed_management_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.field_border_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.field_summary_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.filter_strip_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.forest_f_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.forest_stand_improvement_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.ghg_benefit_measured_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.ghg_benefits_alternate_modeled_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.grassed_waterway_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.hedgerow_planting_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.herbaceous_wind_barriers_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.market_activities_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.mulching_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.nutrient_management_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.partner_activities_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.pasture_and_hay_planting_view.yml
# Only in ./web/modules/custom/cig_pods/config/install: views.view.pods_asset_er.yml
# Only in ./web/modules/custom/cig_pods/config/install: views.view.pods_asset_lists.yml
# Only in ./web/modules/custom/cig_pods/config/install: views.view.pods_csv_exports.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.prescribed_grazingview.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.producer_enrollment_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.project.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.project_summary.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.range_planting_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.residue_and_tillage_management_no_till_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.residue_and_tillage_management_reduced_till_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.riparian_forest_buffer_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.riparian_herbaceous_cover_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.roofs_covers_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.silvopasture_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.stripcropping_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.tree_shrub_establishment_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.vegetative_barrier_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.waste_separation_facility_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.waste_storage_facility_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.waste_treatment_lagoon_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.waste_treatment_view.yml
# Only in ./web/modules/custom/cig_pods_csc/config/install: views.view.windbreak_shelterbelt_establishment_and_renovation_view.yml
# Only in ./web/modules/custom/cig_pods: css
# Only in ./web/modules/custom/cig_pods: img
# Only in ./web/modules/custom/cig_pods: js
# Only in ./web/modules/custom/cig_pods/src: Controller
# Only in ./web/modules/custom/cig_pods/src: Form
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Award.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Awardee.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Contact.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: CostSequence.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: FieldAssessment.php
# Only in ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType: FieldEnrollment.php
# Only in ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType: ImportHistory.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Input.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Irrigation.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: LabResult.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: LabTestingMethod.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: LabTestingProfile.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Operation.php
# Only in ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType: PartnerActivities.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: PastureAssessment.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: PastureHealthAssessment.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: Producer.php
# Only in ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType: ProducerEnrollment.php
diff --color --color -rus ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType/Project.php ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType/Project.php
--- ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType/Project.php	2023-11-03 08:50:25
+++ ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType/Project.php	2023-11-03 08:50:25
@@ -1,15 +1,15 @@
 <?php

-namespace Drupal\cig_pods\Plugin\Asset\AssetType;
+namespace Drupal\cig_pods_csc\Plugin\Asset\AssetType;

 use Drupal\farm_entity\Plugin\Asset\AssetType\FarmAssetType;
 use Drupal\farm_field\FarmFieldFactory;

 /**
- * Provides the Project asset type.
+ * Provides the CIG csc_project asset type.
  *
  * @AssetType(
- * id = "project",
+ * id = "csc_project",
  * label = @Translation("Project"),
  * )
  */
@@ -19,92 +19,110 @@
    * {@inheritdoc}
    */
   public function buildFieldDefinitions() {
+
     $fields = parent::buildFieldDefinitions();

-    // We do not add a "Name" field because we inherit that from the
-    // FarmAssetType class.
     $field_info = [
-      'field_grant_type' => [
-        'type' => 'entity_reference',
-        'label' => 'Grant Type',
-        'target_type' => 'taxonomy_term',
-        'target_bundle' => 'd_grant_type',
-        'handler' => 'default',
-        'required' => FALSE,
-        'multiple' => TRUE,
-          // Lower weight shows up first in form.
-        'weight' => [
-          'form' => 2,
-          'view' => 2,
-        ],
-        'form_display_options' => [
-          'label' => 'inline',
-          'type' => 'options_select',
-        ],
+      'csc_project_id_field' => [
+        'type' => 'string',
+        'label' => 'Project ID',
+        'description' => 'Project ID',
+        'required' => TRUE,
+        'multiple' => FALSE,
       ],
-      'field_funding_amount' => [
-        'label' => 'Funding Amount',
-        'type' => 'fraction',
-        'settings' => [
-          'min' => 0,
-        ],
-        'weight' => [
-          'form' => 3,
-          'view' => 3,
-        ],
-
+      'csc_project_grantee_org' => [
+        'type' => 'string',
+        'label' => 'Grantee Organization Name',
+        'description' => 'Grantee Organization Name',
+        'required' => TRUE ,
+        'multiple' => FALSE,
       ],
-      'field_resource_concerns' => [
-        'type'  => 'entity_reference',
-        'label' => 'Possible Resource Concerns',
-        'target_type' => 'taxonomy_term',
-        'target_bundle' => 'd_resource_concern',
-        'handler' => 'default',
-        'required' => FALSE,
-        'multiple' => TRUE,
-             // Lower weight shows up first in form.
-        'weight' => [
-          'form' => 3,
-          'view' => 3,
-        ],
-        'form_display_options' => [
-          'label' => 'inline',
-          'type' => 'options_select',
-        ],
+      'csc_project_grantee_cont_name' => [
+        'type' => 'string',
+        'label' => 'Grantee Primary Point of Contact',
+        'description' => 'Grantee Primary Point of Contact',
+        'required' => TRUE ,
+        'multiple' => FALSE,
       ],
-      'field_summary' => [
-        'type' => 'string_long',
-        'label' => 'Project Summary',
-        'required' => FALSE,
+      'csc_project_grantee_cont_email' => [
+        'type' => 'string',
+        'label' => 'Grantee Primary Point of Contact Email',
+        'description' => 'Grantee Primary Point of Contact Email',
+        'required' => TRUE ,
         'multiple' => FALSE,
-        'settings' => [
-          'max_length' => 1000,
-          'size' => 60,
+      ],
+      'csc_project_start' => [
+        'type' => 'timestamp',
+        'label' => 'Overall Project Start Date',
+        'description' => 'Overall Project Start Date',
+        'required' => TRUE ,
+        'multiple' => FALSE,
+      ],
+      'csc_project_end' => [
+        'type' => 'timestamp',
+        'label' => 'Overall Project End Date',
+        'description' => 'Overall Project End Date',
+        'required' => TRUE ,
+        'multiple' => FALSE,
+      ],
+      'csc_project_budget' => [
+          'type' => 'fraction',
+          'label' => 'Total Award Budget',
+          'description' => 'Total Award Budget',
+          'required' => TRUE,
+          'multiple' => FALSE,
+      ],
+      'csc_project_comet_version' => [
+        'type' => 'string',
+        'label' => 'COMET-Planner Version',
+        'description' => 'COMET-Planner Version',
+        'required' => TRUE ,
+        'multiple' => FALSE,
+      ],
+      'csc_project_year_reporting' => [
+        'type' => 'list_string',
+        'label' => 'Choose Year of Reporting',
+        'description' => 'Choose Year of Reporting',
+        'allowed_values' => [
+          '2023' => t(string: '2023'),
         ],
-        'form_display_options' => [
-          'label' => 'inline',
-        ],
+        'required' => TRUE ,
+        'multiple' => FALSE,
       ],
-      'award' => [
-        'label' => 'Award Reference',
-        'description' => 'Award Reference',
-        'type' => 'entity_reference',
-        'target_type' => 'asset',
-        'target_bundle' => 'award',
-        'required' => FALSE,
+      'csc_project_month_reporting' => [
+        'type' => 'list_string',
+        'label' => 'Choose Month of Reporting',
+        'description' => 'Choose Month of Reporting',
+        'allowed_values' => [
+          'Jan 1 - March 31' => t(string: 'Jan 1 - March 31'),
+          'April 1 - June 30' => t(string: 'April 1 - June 30'),
+          'July 1 - September 30' => t(string: 'July 1 - September 30'),
+          'October 1 - December 31' => t(string: 'October 1 - December 31'),
+        ],
+        'required' => TRUE ,
         'multiple' => FALSE,
       ],
+      // 'file_upload' => [
+      //     'type' => 'file',
+      //     'label' => 'Excel file to be imported',
+      //     // 'upload_validators' => [
+      //     //   'file_validate_extensions' => [
+      //     //     'png gif jpg',
+      //     //   ],
+      //     // ],
+      // ],
     ];

     $farmFieldFactory = new FarmFieldFactory();
-    foreach ($field_info as $name => $info) {

+    foreach ($field_info as $name => $info) {
       $fields[$name] = $farmFieldFactory->bundleFieldDefinition($info)
         ->setDisplayConfigurable('form', TRUE)
         ->setDisplayConfigurable('view', TRUE);
     }

     return $fields;
+
   }

 }
# Only in ./web/modules/custom/cig_pods_csc/src/Plugin/Asset/AssetType: ProjectSummary.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: RangeAssessment.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: SHMUCropRotation.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: SoilHealthManagementUnit.php
# Only in ./web/modules/custom/cig_pods/src/Plugin/Asset/AssetType: SoilHealthSample.php
Only in ./web/modules/custom/cig_pods_csc/src/Plugin: Log
Only in ./web/modules/custom/cig_pods/src/Plugin: views
Only in ./web/modules/custom/cig_pods/src: ProjectAccessControlHandler.php
Only in ./web/modules/custom/cig_pods: templates
